spec/features/recipes/recipes_page_spec.rb

```diff
@@ -0,0 +1,11 @@
+feature 'user visits recipes page' do
+  before do
+    FactoryBot.create(:recipe, :scrambled_eggs)
+    visit('/')
+  end
+
+  it 'shows user an example data' do
```

Comment:
`shows recipes list`. Part `example data` does not really tell anything because it is very generic, you could write description like this for 90% of you intergation tests ;).